### PR TITLE
Getting captions

### PR DESCRIPTION
**Pull request by Acorde**

### CODE REVIEW

File: src/main/java/com/applicaster/jwplayerplugin/JWPlayerUtil.java [hunk 1]
```diff
@@ -55,22 +55,35 @@ public static PlaylistItem getPlaylistItem(Playable playable, Map pluginConfigur
...
+            if (tracks != null) {
+                for (int i = 0; i < tracks.size(); i++) {
+                    LinkedHashMap<String, String> sideCarCaption = (LinkedHashMap<String, String>) tracks.get(i);
+
+                    String src = sideCarCaption.get("source");
+                    String label = sideCarCaption.get("label");
+
+
+                    if (src != null) {
+                        Caption caption = new Caption();
```
Comment (guysarfatty) [inline src/main/java/com/applicaster/jwplayerplugin/JWPlayerUtil.java]: Please use CaptionBuilder to create the new caption

File: src/main/java/com/applicaster/jwplayerplugin/JWPlayerUtil.java [hunk 2]
```diff
@@ -55,22 +55,35 @@ public static PlaylistItem getPlaylistItem(Playable playable, Map pluginConfigur
...
+
+            LinkedHashMap textTracks = (LinkedHashMap) ((APAtomEntry.APAtomEntryPlayable) playable).getEntry().getExtensions().get("text_tracks");
+
+            if (textTracks != null) {
+                tracks = (ArrayList) textTracks.get("tracks");
+            }
+
+            if (tracks != null) {
+                for (int i = 0; i < tracks.size(); i++) {
+                    LinkedHashMap<String, String> sideCarCaption = (LinkedHashMap<String, String>) tracks.get(i);
```
Comment (guysarfatty) [inline src/main/java/com/applicaster/jwplayerplugin/JWPlayerUtil.java]: Let's rename sideCarCatption to textTrack, as it is more precise name for it

File: src/main/java/com/applicaster/jwplayerplugin/JWPlayerUtil.java [hunk 3]
```diff
@@ -55,22 +55,35 @@ public static PlaylistItem getPlaylistItem(Playable playable, Map pluginConfigur
...
+
+                    String src = sideCarCaption.get("source");
+                    String label = sideCarCaption.get("label");
+
```
Comment (guysarfatty) [inline src/main/java/com/applicaster/jwplayerplugin/JWPlayerUtil.java]: Please also obtain the "kind" parameter , and pass it to the CaptionBuilder.

Comment (Acorde) [reply src/main/java/com/applicaster/jwplayerplugin/JWPlayerUtil.java]: The setKind method is receiving  CaptionType enum which has the following params:
CAPTIONS,
CHAPTERS,
THUMBNAILS;

Here is the caption Data:
```
tracks: [
{
label: "Românesc",
kind: "subtitles",
type: "vtt",
language: "ro",
source: "https://mediafiles.cms.protvplus.ro/files/2019/08/29/a30b6e79-5b82-4e21-a6c7-607a8aa672dc-vtt_subtitles.vtt"
}
]
```

What are the kinds the data will hold and which kind will fit the CaptionType enum parameters?

**Pull request merged**